packages/build-tools/src/utils/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const IOS_CACHE_KEY_PREFIX = 'ios-ccache-';
+export const IOS_CACHE_PATH = 'Library/Caches/ccache';
```

Comment:
nit: I don't like those are relative paths, but it's not necessarily clear they should be resolved from `$HOME`. Also, technically, whether one should use `IOS_CACHE_PATH` or `ANDROID_CACHE_PATH` does not depend on `ctx.job.platform`, but on `os.platform()`. Can we change this accordingly? Thinking of a `getCcacheCacheDirectory({ env }) { os.platform() ? ... : ... }`